### PR TITLE
Add subscription metrics

### PR DESCRIPTION
**Pull request by gsvr**
- [x] subscriptions.total.last
- [x] subscriptions.completed.last
- [x] subscriptions.broken.last

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: subscriptions/tasks.py [hunk 1]
```diff
@@ -329,3 +329,52 @@ def run(self):
...
+
+
+class FireCreatedLast(Task):
+
+    """ Fires last created subscriptions count
+    """
+    name = "seed_staged_based_messaging.subscriptions.tasks.fire_created_last"
+
+    def run(self):
+        created_subs = Subscription.objects.filter()
```
Comment (imsickofmaps) [inline subscriptions/tasks.py]: `created_subs = Subscription.objects.all().count()` please

File: subscriptions/tasks.py [hunk 2]
```diff
@@ -329,3 +329,52 @@ def run(self):
...
+
+
+class FireBrokenLast(Task):
+
+    """ Fires last broken subscriptions count
+    """
+    name = "seed_staged_based_messaging.subscriptions.tasks.fire_broken_last"
+
+    def run(self):
+        broken_subs = Subscription.objects.filter(process_status=-1)
```
Comment (imsickofmaps) [inline subscriptions/tasks.py]: just do `.count()` on all of them

**Comment by imsickofmaps** [comment]
👍 tweak the land when travis happy

**Pull request merged**